src/ucp/wireup/address.c

```diff
@@ -312,16 +312,57 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
     }
 }
 
+static uint32_t ucp_address_pack_flags(uint64_t input_flags,
+                                       uint64_t cap_flags,
+                                       uint8_t output_start_bit)
+{
+    uint32_t result_flags = 0;
+    uint32_t packed_flag;
+    uint8_t cap_index;
+
+    ucs_assert(output_start_bit < 32);
```

Comment:
ucs_assert((ucs_popcount(cap_flags) + output_start_bit) < 32)

Reply:
yes, makes sense to have this check